confluent_kafka/src/confluent_kafka.h

```diff
@@ -64,8 +64,15 @@
 
 /**
  * @returns Unicode Python object as char * in UTF-8 encoding
+ * @param uobjp might be set to NULL or a new object reference (depending
```

Comment:
Along the same lines as my other comment -- why not `Py_XDECREF` immediately inside these methods? Doesn't seem like there would ever be a case where we'd want to save this intermediate data, especially since it is only an issue in Py2..

Reply:
I think this comment is misleading, it is not a new object reference to the original object, but a new object reference to an utf8 reprsentation object of the original object, do you object?